### ISSUE TITLE
AIP-38 | List Variables

### ISSUE THREAD

**Issue by bbovenzi** [apache/airflow#43709]
Use the auto-generated list variables query to display a table of variables. The page should be accessed in the nav via Admin -> Variables. It should use `DataTable` to handle pagination and sorting.

**Comment by shubhamraj-git** [comment]
Hey @bbovenzi , Could you assign this to me?
Also do we have any pre-screenshots? or any pre-discussed idea?

**Comment by bbovenzi** [comment]
@shubhamraj-git All yours. I think you can check out how the Events page at "Browse -> Events" works. The designs should basically be covered by our DataTable component.

**Comment by bbovenzi** [comment]
Yes, let's get the list done first and then do others in subsequent PRs

**Comment by shubhamraj-git** [comment]
I think we can move this to Done, All the major items are completed.
If something comes up, can be treated as bug.
cc: @jscheffl @pierrejeambrun @bbovenzi

**Comment by pierrejeambrun** [comment]
Sounds good to me, closed.
